packages/common/src/chains/mudFoundry.ts

```diff
@@ -3,7 +3,7 @@ import { MUDChain } from "./types";
 
 export const mudFoundry = {
   ...foundry,
-  fees: {
-    defaultPriorityFee: 0n,
-  },
+  // fees: {
+  //   defaultPriorityFee: 0n,
+  // },
```

Comment:
Did you encounter serialization errors with this? I think @yonadaaa saw a similar thing.